db/smt-db

```diff
@@ -304,7 +305,7 @@ sub setup
     my ($login,$pass,$uid,$gid) = getpwnam("wwwrun");
     chown 0, $gid, "/etc/smt.conf";
 
-    `smt-sql --select-mode-direct /usr/share/schemas/smt/mysql/2.03/100-smt-tables.sql > /var/log/smt/smt-db-setup.log`;
+    `smt-sql --select-mode-direct /usr/share/schemas/smt/mysql/3.00/100-smt-tables.sql > /var/log/smt/smt-db-setup.log`;
```

Comment:
Hmm, the old is wrong but the new one is not even better :-)

I think we should create a "latest" directory".
Then we have only the "latest" and the migration directories "<oldversion>-<newversion>" .

A new setup happens always with the "latest" version and no migration is needed in this case.
Schema migrations are only for the running system.

@aaannz : What do you think?